ui/panels-plugin/src/plugins/stat-chart/utils/data-transform.ts

```diff
@@ -38,6 +38,20 @@ export function convertSparkline(
       }
     });
   }
+  return color;
+}
+
+export function convertSparkline(
+  chartsTheme: PersesChartsTheme,
+  sparkline?: SparklineOptions,
+  thresholds?: ThresholdOptions,
+  value?: number
+): LineSeriesOption | undefined {
+  if (sparkline === undefined) return;
+
+  // TO DO: add option for color scheme? Should default color derive from threshold.defaultColor or sparkline.color?
+  const defaultColor = chartsTheme.thresholds.defaultColor ?? chartsTheme.sparkline.color;
```

Comment:
Not sure I 100% understand the question in the TODO, I guess it depends if thresholds in panel spec is defined or not right? If it’s undefined, always go with chartsTheme.sparkline.color for default?

Reply:
I noticed this case when testing and found it kind of confusing. Guessing that's what Eunice is describing here. 

<img width="903" alt="image" src="https://user-images.githubusercontent.com/396962/230680680-e06d77b4-ac4e-4593-85a3-838b144b7cc8.png">

Reply:
In grafana there is an option for color scheme. I think if you choose "from thresholds", default color is using chartsTheme.threshold.defaultColor, or panel spec `thresholds.default_color`  even when you don't have threshold defined.

![image](https://user-images.githubusercontent.com/9817826/230680603-59a9c448-2bd2-4a89-89f8-2c9380a57bf2.png)

Reply:
This might be a UX that's worth talking through with one of the designers. It is a little weird to set the default line color by setting the default threshold, but it's also really weird to have the line and the text be different colors in this case.

Reply:
@juliepagano ooh how did you get that? are you using `sparkline.color`? 

there are too many different ways you can set sparkline colors lol 🤯

Reply:
I also think I maybe found a bug while I was in here. It looks like editing the color of the default threshold doesn't work if you haven't added other threshold values. I wonder if we shouldn't show/use the default value at all unless a threshold is set?

Reply:
I was also able to hit a case where I modified the default value, but the text was still using the "default" green default threshold value 

<img width="900" alt="image" src="https://user-images.githubusercontent.com/396962/230681307-35cac92b-5993-4949-8175-69884eebf699.png">

Reply:
> @juliepagano ooh how did you get that? are you using `sparkline.color`?
> 
> there are too many different ways you can set sparkline colors lol 🤯

That was the default state of one of the perses app panels when I ran your PR. It's one of the stat charts in the first row if you go to http://localhost:3000/projects/perses/dashboards/Demo?start=5m&var-instance=demo.do.prometheus.io%3A9100&var-interval=1m

Reply:
oh my LOL oops it definitely wasn't like that earlier, i must have broke something 😆